
// SPDX-License-Identifier: MIT
// Copyright (c) 2024 nyashbox

syntax = "proto3";

package guidemyride.services.driver;

message StubRequest {
	string request = 1;
}

message StubResponse {
	string response = 1;
}

service DriverService {
	rpc StubMethod(StubRequest) returns (StubResponse);
}
